src/app/commands/cmd_cel_properties.cpp

```diff
@@ -126,9 +126,25 @@ class CelPropertiesWindow : public app::gen::CelProperties
 
       case kKeyDownMessage:
         if (opacity()->hasFocus()) {
-          if (static_cast<KeyMessage*>(msg)->scancode() == kKeyEnter) {
-            onCommitChange();
-            closeWindow(this);
+          bool isKeyUsed = false;
+
+          switch (static_cast<KeyMessage*>(msg)->scancode()) {
+
+            case kKeyEnter:
+              onCommitChange();
+              closeWindow(this);
+              isKeyUsed = true;
+              break;
+
+            case kKeyEsc:
+              closeWindow(this);
```

Comment:
I think the `kKeyEsc` should call `onCommitChange()` too, because it acts like a shortcut to close the dialog, but not to cancel the changes in the dialog (e.g. we quickly change the layer name and press `Esc` doesn't mean that we want to discard the layer name change).

I think esc should act exactly like `kKeyEnter`. So this could be just:

    case kKeyEnter:
    case kKeyEsc:
       onCommitChange();
       closeWindow(this)...